components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/OAuthAdminService.java

```diff
@@ -675,7 +688,7 @@ public OAuthRevocationResponseDTO revokeAuthzForAppsByResoureOwner(
                     if (appDTO.getApplicationName().equals(appName)) {
                         Set<AccessTokenDO> accessTokenDOs;
                         try {
-                            // retrieve all ACTIVE or EXPIRED access tokens for particular client authorized by this user
+                            //retrieve all ACTIVE or EXPIRED access tokens for particular client authorized by this user
```

Comment:
You should have a space before starting the comment. Is there any specific reason to remove this space?

Reply:
fixed with https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/942/commits/ca4690d734709b108ef2df33a69ae3c9242febc8